roles/mockturtle/files/sopel_modules/invite.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# vi: set fileencoding=utf-8
+
+import time
+import sopel.bot as bot
+import sopel
+
+@sopel.module.rule(r'.*[Ii]nvite.*')
+@sopel.module.rate(900)
+def invite(bot, trigger):
+    """To new members that are looking for an invitation"""
+    joinedTime = bot.db.get_nick_value(trigger.nick,
+                                       'join_channel_' + trigger.sender)
+    diff = time.gmtime() - joinedTime
+    if diff >= 3600:
+        return
+    bot.reply("If you would like an invite to lobste.rs, please look "
+              "at the chat FAQ first. Thank you. https://lobste.rs/chat "
+              "(I am a bot. (v).v.(v))")
+
+@sopel.module.event('JOIN')
+@priority('low')
```

Comment:
If you run a Python interpreter over this file, you'll find the following error:

    python invite.py
    Traceback (most recent call last):
      File "invite.py", line 22, in <module>
        @priority('low')
    NameError: name 'priority' is not defined

The priority (and unblockable, on the line below) are in the sopel.module package.  Like `sopel.module.event` on the line above, these two lines should include `sopel.module.`

Reply:
That's embarrasing. Pushing a fix now.